src/main/java/io/github/protonmc/proton/mixin/client/entities/LivingEntityRendererMixin.java

```diff
@@ -26,14 +26,14 @@
 @Mixin(LivingEntityRenderer.class)
 public abstract class LivingEntityRendererMixin<T extends LivingEntity, M extends EntityModel<T>> extends EntityRenderer<T> implements FeatureRendererContext<T, M> {
     public LivingEntityRendererMixin(EntityRenderDispatcher dispatcher) {
-		super(dispatcher);
+        super(dispatcher);
     }
-    
+
     @Unique
     private float creeperColor = 1.0F;
 
     @Inject(method = "render", at = @At("HEAD"))
-	public void getCreeperColor(T livingEntity, float f, float g, MatrixStack matrixStack, VertexConsumerProvider vertexConsumerProvider, int i, CallbackInfo ci) {
+    public void getCreeperColor(T livingEntity, float f, float g, MatrixStack matrixStack, VertexConsumerProvider vertexConsumerProvider, int i, CallbackInfo ci) {
```

Comment:
Huh?

Reply:
this was an older commit. i forgot to remove it